docs/Backend/Routes/user_routes.md

```diff
@@ -1,70 +1,46 @@
-# User's routes
+# Customer's routes
 
-### Get all users 
+### Get all customers 
 - Method: **GET**
-- Endpoint: `/user`
+- Endpoint: `/customer`
 - Authorization: Bearer token given from login
 
-### Get a single user by ID
+### Get a single customer by ID
 - Method: **GET**
-- Enpoint: `/user/:id`
+- Enpoint: `/customer/:customerID`
 - Authorization: Bearer token given from login
 
-### Login
+### Create a customer
 - Method: **POST**
-- Endpoint: `/user/login`
-- Authorisation: Basic email - password
-
-### Create a user
-- Method: **POST**
-- Endpoint: `/user`
+- Endpoint: `/customer`
 - Authorization: Bearer token given from login
 - Payload:
     ```JSON
     {
-        "name": "New User",
-        "email": "newEmail@email.com",
-        "role": "Admin",
-        "password": "SuperSecretPassword",
+        "name": "New Customer",
+        "email": "newEmail@email.com"
     }
     ```
 
-### Send a recovery email
-- Method: **POST**
-- Endpoint: `/user/forgot`
-- Payload:
-    ```JSON
-    {
-        "email": "newEmail@email.com",
-    }
-    ```
-
-### Modify the password
-- Method: **POST**
-- Endpoint: `/user/reset`
-- Authorisation: Basic emailToken - newPassword
-
-### Update a user
+### Update a customer
 - Method: **PUT**
-- Endpoint: `/user/:id`
+- Endpoint: `/customer/:customerID`
 - Authorization: Bearer token given from login
 - Payload:
     ```JSON
     {
         "id" : 1,
-        "name": "New User",
-        "email": "newEmail@email.com",
-        "role": "Admin",
-        "password": "SuperSecretPassword",
+        "name": "New Customer",
+        "email": "newEmail@email.com"
     }
     ```
 
-### Delete a user
-- Method: **DELETE**
-- Endpoint: `/user/:id`
+### Archive or un-archive a customer
+- Method: **POST**
+- Endpoint: `/customer/archive`
 - Authorization: Bearer token given from login
 
-### Delete **ALL** users
-- Method: **DELETE**
-- Endpoint: `/user`
+### Get **ALL** archived or non-archived customers
+- Method: **GET**
+- Endpoint: `/customer/archive/:archived`
 - Authorization: Bearer token given from login
```

Comment:
This should be in the customer routes not in the user_routes.md

Reply:
Oops, mixed up the file names.